Resources/Prototypes/Maps/angle.yml

```diff
@@ -0,0 +1,68 @@
+type: gameMap
+  id: Angle
+  mapName: 'Angle'
+  mapPath: /Maps/angle.yml
+  minPlayers: 35
+  maxPlayers: 80
+  stations:
+    Angle:
+      stationProto: StandardNanotrasenStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Angle {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath:
```

Comment:
Empty shuttle path

Reply:
i'll wait for you to port over the nyano one

Reply:
It's been ported